easybuild/tools/github.py

```diff
@@ -360,14 +383,16 @@ def fetch_easyconfigs_from_pr(pr, path=None, github_user=None):
     last_commit = commits_data[-1]
     _log.debug("Commits: %s, last commit: %s" % (commits_data, last_commit['sha']))
 
-    # obtain most recent version of patched files
-    for patched_file in patched_files:
-        # path to patch file, incl. subdir it is in
-        fn = os.path.sep.join(patched_file.split(os.path.sep)[-2:])
-        sha = last_commit['sha']
-        full_url = URL_SEPARATOR.join([GITHUB_RAW, GITHUB_EB_MAIN, GITHUB_EASYCONFIGS_REPO, sha, patched_file])
-        _log.info("Downloading %s from %s" % (fn, full_url))
-        download_file(fn, full_url, path=os.path.join(path, fn), forced=True)
+    if not stable or closed:
+        print "WARNING: Using unstable/closed PR on easybuild installation!"
```

Comment:
the `on easybuild installation` is a bit weird, maybe go with `\n*** WARNING: Using easyconfigs from unstable/closed PR #<id>***\n` (where you replace `<id>` with the PR ID)